src/main/java/com/microsoft/azure/kusto/kafka/connect/sink/KustoSinkConfig.java

```diff
@@ -1,5 +1,8 @@
 package com.microsoft.azure.kusto.kafka.connect.sink;
 
+import com.fasterxml.jackson.core.JsonParser;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
```

Comment:
Since we have touched this file and this is a major release , KUSTO_SINK_ENABLE_TABLE_VALIDATION can be defaulted to false. The validation of ingestion role can be an opt-in. This was not a fallout of this PR , something we wanted to hold on for a new release :) Since you have already done this contrib, we can change the default and add this in the documentation

Reply:
Did the change and updated the documentation